### PR TITLE
Bug/remove survey review submit button

### PR DESCRIPTION
**Pull request by ElijahRW**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
   * Removed Submit button from Appoitnment Review
   * Added Update Notes Button
   * Updated aud-navigation to display `Patient Notes`

### CODE REVIEW

**Review by Liriaene — APPROVED** [review]
Changes look good to me.

File: client/src/app/notes/notes.component.ts [hunk 1]
```diff
@@ -24,13 +26,18 @@ export class NotesComponent implements OnInit {
         });
     }
     // calls the updateNotes service from patientAdminService
-    public submitNote(patinetID: number): void {
+    public submitNote(patinetID: number, displayPopup?: boolean): void {
         if (typeof (this.content) === 'string') {
             this.adminPatientService.updateNotes(patinetID, this.content).subscribe((_) => {
-                console.log('Notes Updated Successfully');
-            }, (error) => {
```
Comment (wynnliam) [inline client/src/app/notes/notes.component.ts:31]: Wouldn't it be safe to keep the error handling code here?

**Review by wynnliam — APPROVED** [review]
Everything seems good to me. I had a slight concern about nixing some error handling code. However, it's not anything to block an approval in my opinion.

File: client/src/app/notes/notes.component.ts (hunk 1 above)
Comment (ElijahRW) [reply client/src/app/notes/notes.component.ts:31]: This subscribe doesn’t need to handle errors. In fact, all it was doing was writing to the console: we should only handle errors here if we want a specific reaction on the front end.

Comment (ElijahRW) [reply client/src/app/notes/notes.component.ts:31]: This subscribe doesn’t need to handle errors. In fact, all it was doing was writing to the console: we should only handle errors here if we want a specific reaction on the front end.

**Pull request merged**